keps/sig-storage/2451-service-account-token-volumes/kep.yaml

```diff
@@ -0,0 +1,21 @@
+title: Service Account Token Volumes
+kep-number: 2451
+authors:
+  - "@smarterclayton"
+  - "@liggitt"
+  - "@mikedanese"
+  - "@zshihang"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-auth
+reviewers:
+  - "@jsafrane"
+  - "@tallclair"
+approvers:
+  - "@saad-ali"
+editor: "@zshihang"
+creation-date: 2018-05-15
+last-updated: 2020-03-27
+status: implemented
```

Comment:
I believe implemented is used when after the feature is GA. Since this is still alpha (targeting beta), this should be "implementable"

Reply:
Actually sorry, I was mixing this up with https://github.com/zshihang/enhancements/blob/fe2e7486ee658552445edcff3f4be7b3e25fd535/keps/sig-storage/1855-csi-driver-service-account-token/README.md.

@zshihang can you confirm the feature phases and release milestones?

Reply:
this is GAed in 1.20 as TokenRequestProjection

Reply:
So is it fine to have it in "implemented" status ?

Reply:
I think update on KEP can be done after the PR merge.

Reply:
@msau42 - if it's implemented, I suggest merging as is now
[we can extend exact milestones if needed later - I don't think we need to backfill all the data everywhere]

Reply:
Sorry for the confusion, this is fine, thanks!